packages/webpack-config/tests/__tests__/basic-test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+// const puppeteer = require('puppeteer');
+const path = require('path');
```

Comment:
It should be possible to use ES6 imports in Jest tests.

Reply:
https://circleci.com/gh/expo/expo-cli/3649?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link